tests/DisplayDataTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+require_once '../src/DisplayData.php';
+require_once '../src/Team.php';
```

Comment:
Should be able to just require the autoload file here instead of these individually.